tests/deb/control_field_test.py

```diff
@@ -52,23 +52,23 @@ def test_multiline(self):
     self.assertEqual(
         'Description: fizzbuzz\n',
         make_deb.MakeDebianControlField(
-            'Description', 'fizzbuzz', is_multiline=True))
+            'Description', 'fizzbuzz', multiline=make_deb.Multiline.YES))
```

Comment:
The tests  should all be for the format we actually use, YES_ADD_NEWLINE.

Reply:
We use both. Admittedly this should also include a case for `YES_ADD_NEWLINE` but that is already tested by the changes file test.

Reply:
I don't see any calls to MakeDebianControlField that use YES instead of YES_AND_ADD_NEWLINE.

Reply:
In `CreateDebControl`
`multiline = Multiline.YES if values[2] else Multiline.NO` will be `Multiline.YES` for `Description` and will call `controlfile += MakeDebianControlField(fieldname, kwargs[key], multiline)`